packages/elements-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stoplight/elements-core",
-  "version": "7.9.0",
+  "version": "7.9.1",
```

Comment:
I'm not strongly opinionated about this but I believe this is a `MINOR` change rather than `PATCH`. It was never stated that `Elements` design is responsive, and while this might not be a typical feature, it still improves the usage.

Reply:
I agree and disagree in the same time, but also don't have strong opinion on that. Maybe we could have it as minor and all other responsive related releases treat as patches.

Reply:
I've make it a minor version change for elements-core and elements. Elements-dev-portal has minor version bumped already by @kaylachun with search component.